src/widgets/list/lv_list.h

```diff
@@ -17,6 +17,10 @@ extern "C" {
 
 #if LV_USE_LIST
 
+#if LV_USE_FLEX == 0
+#error "LV_LIST requires LV_USE_FLEX to be enabled"
```

Comment:
Actually we have this format for [other widgets](https://github.com/lvgl/lvgl/blob/master/src/widgets/slider/lv_slider.h#L22)
```suggestion
#error "lv_list: lv_flex is required. Enable it in lv_conf.h (LV_USE_FLEX 1)"
```

Reply:
Fair enough! I was copying the style from the lv_menu error rather than the lv_list one. I've updated so those three match. When I get a chance I'll have a look to see if there's other errors now which are mismatched.